tests/Aspire.Hosting.Keycloak.Tests/KeycloakResourceBuilderTests.cs

```diff
@@ -144,4 +145,108 @@ public async Task VerifyManifest()
             """;
         Assert.Equal(expectedManifest, manifest.ToString());
     }
+
+    [Fact]
+    public void WithReverseProxyAddsEnvironmentVariables()
```

Comment:
These test should use GetEnvironmentVariableValuesAsync and not mess with annotations directly.